Makefile.am

```diff
@@ -1680,18 +1680,31 @@ dist_man8_MANS += \
 	man/httpd.8
 endif # HTTPD
 
-if BENCH
-check_PROGRAMS += bench/cyrdbbench
-bench_cyrdbbench_SOURCES = bench/cyrdbbench.c imap/mutex_fake.c
-bench_cyrdbbench_LDADD = $(LD_BASIC_ADD)
-endif # BENCH
-
 if REPLICATION
 dist_man8_MANS += \
 	man/sync_client.8 \
 	man/sync_reset.8 \
 	man/sync_server.8
-endif
+endif # REPLICATION
+
+if MAINTAINER_MODE
```

Comment:
The `if MAINTAINER_MODE` shall be removed and the files moved to the dist_man8_MANS variable on line 1627.

The conditionals from lines 1678, 1672 and others shall be removed.  As a rule of thumb, it makes no sense to have a `dist_` prefix within Automake conditional.

Reply:
The conditional lines need to be conditional to prevent the man pages being _installed_ for features that haven't been built.

The conditionalised lines could equally be `man8_MANS` (without the `dist_`) in this case, but that would mean more changes beyond what's necessary, and more testing.

Our plan for some point in the future is that none of these features will be optional anymore, and at that point all this conditionalisation will go away, so it's not worth the work right now to tinker with it beyond just fixing the bugs.